assets/src/components/nav/navMenu.tsx

```diff
@@ -85,6 +88,22 @@ const NavMenu: React.FC<Props> = ({ mobileMenuIsOpen, toggleMobileMenu }) => {
                 <BsIcon.GearFill /> Settings
               </Nav.Link>
             </Nav.Item>
+            {keycloakEnabled && (
+              <>
+                <Nav.Item>
+                  <hr />
+                </Nav.Item>
+                <Nav.Item>
+                  <Nav.Link
+                    className="icon-link"
+                    as="a"
+                    href="/auth/keycloak/logout"
+                  >
+                    <BsIcon.BoxArrowRight /> Logout
```

Comment:
**suggestion:** the [Bootstrap `icon-link` docs](https://getbootstrap.com/docs/5.3/helpers/icon-link/) recommend setting `aria-hidden` on the icon in a case like this where the icon is decorative and doesn't provide additional information (I guess this applies to the other entries as well and I just didn't catch it in the previous PR).

Reply:
Thanks for catching that! d59256c671021b2643d4cb3b949ca9fb6bbc54b6
https://github.com/mbta/skate/pull/2391#discussion_r1489811307